### ISSUE TITLE
New command to find modules in bicep registry `Find-BicepModule`

### ISSUE THREAD

**Issue by StefanIvemo** [PSBicep/PSBicep#241]
**Is your feature request related to a problem? Please describe.**
We need a command that helps the user find available modules in Bicep Registries. The command should be able to search for modules in an ACR and look provide info about modules used in a specific bicep file.

**Describe the solution you'd like**

```powershell
# Finds all ACR referenced in the current template and searches for available modules.
Find-BicepModule -Path main.bicep

# Finds all ACRs in the local cache and searches for available modules
Find-BicepModule -Cache

# Finds all modules in a specific ACR (This should support a module registry alias from `bicepconfig.json` as well
Find-BicepModule -Registry psbicep.azurecr.io
```

**Additional context**
New functionality must be added to BicepNet to support this.

**Comment by StefanIvemo** [comment]
@PalmEmanuel does this look ok? Did I get it right?

**Comment by PalmEmanuel** [comment]
Looks about right! I forgot about the specific `-Registry` use case, but I'll see if I can add it! Currently the parameters are `-Path` and `-UseCache` (can be changed to `-Cache`).

I've built it so that they can be combined, so you can point to a file and also include the cache (in case the file wasn't restored for example), do we want to add the `-Registry` in a similar way, or which parameters should be allowed together?

**Comment by PalmEmanuel** [comment]
Currently I've just added all the endpoints together and searched through them, then output the results in a list, but you imply a valid point that there could be value to separating the results for clarity between file and cache (and registry).

**Comment by PalmEmanuel** [comment]
The parameter `-Registry` is now added, and I changed the parameters so they're no longer possible to combine since I didn't find a clean way to present the difference. [PR #16 in BicepNet](https://github.com/PSBicep/BicepNet/pull/16)

**Comment by StefanIvemo** [comment]
`Find-BicepNetModule` and `Get-BicepNetCachePath` are now available in BicepNet v2.0.2. Removing the `dependency: BicepNet` label.
